src/theory/bv/bv_eager_solver.cpp

```diff
@@ -59,81 +58,79 @@ void EagerBitblastSolver::initialize() {
 #endif
   } else {
     d_bitblaster = new EagerBitblaster(d_bv);
-    THEORY_PROOF(
-      if( d_bvp ){
-        d_bitblaster->setProofLog( d_bvp );
-        d_bvp->setBitblaster(d_bitblaster);
-      }
-    );
+    THEORY_PROOF(if (d_bvp) {
+      d_bitblaster->setProofLog(d_bvp);
+      d_bvp->setBitblaster(d_bitblaster);
+    });
   }
 }
 
 bool EagerBitblastSolver::isInitialized() {
-  bool init = d_aigBitblaster != NULL || d_bitblaster != NULL;
-  if (init) {
-    Assert (!d_useAig || d_aigBitblaster);
-    Assert (d_useAig || d_bitblaster);
-  }
+  const bool init = d_aigBitblaster != nullptr || d_bitblaster != nullptr;
+  Assert(!init || !d_useAig || d_aigBitblaster);
+  Assert(!init || d_useAig || d_bitblaster);
   return init;
 }
 
 void EagerBitblastSolver::assertFormula(TNode formula) {
   d_bv->spendResource(1);
-  Assert (isInitialized());
-  Debug("bitvector-eager") << "EagerBitblastSolver::assertFormula "<< formula <<"\n"; 
+  Assert(isInitialized());
+  Debug("bitvector-eager") << "EagerBitblastSolver::assertFormula " << formula
+                           << "\n";
   d_assertionSet.insert(formula);
-  //ensures all atoms are bit-blasted and converted to AIG
-  if (d_useAig)
-  {
+  // ensures all atoms are bit-blasted and converted to AIG
+  if (d_useAig) {
 #ifdef CVC4_USE_ABC
     d_aigBitblaster->bbFormula(formula);
 #else
     Unreachable();
 #endif
-  }
-  else
+  } else {
     d_bitblaster->bbFormula(formula);
+  }
 }
 
 bool EagerBitblastSolver::checkSat() {
-  Assert (isInitialized());
-  std::vector<TNode> assertions; 
-  for (AssertionSet::const_iterator it = d_assertionSet.begin(); it != d_assertionSet.end(); ++it) {
-    assertions.push_back(*it); 
+  Assert(isInitialized());
+  std::vector<TNode> assertions;
+  for (TNode assertion : d_assertionSet) {
+    assertions.push_back(assertion);
   }
-  if (!assertions.size())
+
+  if (assertions.empty()) {
```

Comment:
Since you are already cleaning up this code, maybe also move this before copying the assertions and just check if the set is empty?

Reply:
Done.